pkg/model/storage/health.go

```diff
@@ -74,22 +74,22 @@ func (s *storeHealthTracker) setDatabaseVersion(version byte) error {
 	return nil
 }
 
-func (s *storeHealthTracker) checkCorrectDatabaseVersion(expectedVersion byte) (bool, error) {
+func (s *StoreHealthTracker) CheckCorrectDatabaseVersion() (bool, error) {
 
 	value, err := s.store.Get([]byte("dbVersion"))
 	if err != nil {
 		return false, errors.Wrap(NewDatabaseError(err), "failed to read database version")
 	}
 
 	if len(value) > 0 {
-		return value[0] == expectedVersion, nil
+		return value[0] == DBVersion, nil
```

Comment:
Thing is we currently do not have a need for versioning and migrations. I'd keep it like that until we really need to migrate something

Reply:
alright, was just wondering because why should the main database version be increased, if we change something for example in the participation database layout :)